src/charm.py

```diff
@@ -1,34 +1,146 @@
 #!/usr/bin/env python3
 # Copyright 2021 Canonical Ltd.
 # See LICENSE file for licensing details.
-
+import yaml
+import json
 import logging
+import ops.charm
 import subprocess
+from subprocess import check_call
 from urllib.request import urlopen
 
-from ops.charm import CharmBase
 from ops.main import main
-from ops.model import BlockedStatus, MaintenanceStatus, WaitingStatus
+from ops.model import BlockedStatus, MaintenanceStatus, WaitingStatus, ActiveStatus, Relation
 from charms.operator_libs_linux.v0 import apt
+from charms.operator_libs_linux.v0.systemd import service_restart, service_resume
+from mongoserver import MongoDB, MONGODB_PORT
+from typing import Optional
 
 logger = logging.getLogger(__name__)
 
+PEER = "mongodb"
+
 
-class MongodbOperatorCharm(CharmBase):
+class MongodbOperatorCharm(ops.charm.CharmBase):
```

Comment:
Seems like an odd change? Nothing *wrong* as such though

Reply:
yeah, tbh not sure why I did this